recipe/bld.bat

```diff
@@ -3,7 +3,7 @@ REM Install headers with projectConfig.cmake files with cmake
 mkdir build
 cd build
 cmake -G "NMake Makefiles" ^
-      -D CMAKE_INSTALL_PREFIX=%PREFIX% ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
```

Comment:
Can you try to put the argument in quotes?

Reply:
Also try setting `-D CMAKE_INSTALL_INCLUDEDIR:FILEPATH="%LIBRARY_PREFIX%\include"` from what I see in `CMakeLists.txt`

Reply:
`CMAKE_INSTALL_INCLUDEDIR` is set properly by `GNUInstallDirs` and defaults to `${CMAKE_INSTALL_PREFIX}/include` on Windows. The problem is that this variable is overwritten in the CMakeLists:

```cmake
option (USE_PYTHON_INCLUDE_DIR "Install pybind11 headers in Python include directory instead of default installation prefix" OFF)
if (USE_PYTHON_INCLUDE_DIR)
    file(RELATIVE_PATH CMAKE_INSTALL_INCLUDEDIR ${CMAKE_INSTALL_PREFIX} ${PYTHON_INCLUDE_DIRS})
endif()
```

Turning `USE_PYTHON_INCLUDE_DIR=OFF` solves the problem (tested locally). The documentation says "Install pybind11 headers in Python include directory instead of default installation prefix" and is `OFF` by default. Any idea why it is `ON` in the recipe?

Reply:
No idea, could be because the PyPI package is installed that way.